examples/kotlin-multiplatform/build.gradle

```diff
@@ -1,17 +1,55 @@
 import kotlinx.benchmark.gradle.JsBenchmarksExecutor
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
 
 plugins {
     id 'org.jetbrains.kotlin.multiplatform'
     id 'org.jetbrains.kotlin.plugin.allopen' version "1.9.21"
     id 'org.jetbrains.kotlinx.benchmark'
+    id 'com.android.library'
 }
 
 // how to apply plugin to a specific source set?
 allOpen {
     annotation("org.openjdk.jmh.annotations.State")
 }
 
+android {
+    compileSdk 34
+    namespace = "org.jetbrains.kotlinx.examples"
+
+    defaultConfig {
+        minSdk = 29
+        targetSdk = 34
+        versionCode = 1
+        versionName = "1.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+//            isMinifyEnabled = false
+//            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+}
+
+// Could not determine the dependencies of task ':examples:kotlin-multiplatform:extractReleaseAnnotations'.
+// > Could not resolve all task dependencies for configuration ':examples:kotlin-multiplatform:detachedConfiguration1'.
+//    > Could not find com.android.tools.lint:lint-gradle:31.2.2.
+repositories {
+    google()
+}
+
 kotlin {
+    androidTarget {
+        // Android target does not have any compilations
+        println("android compilations: ${compilations.size()}")
+    }
```

Comment:
JFYI: when `implementation project(":kotlinx-benchmark-runtime")` is replaced with a dependency on actual artifact (for instance, published to a local maven repo), `:compileReleaseKotlinAndroid` fails due to unresolved `kotlinx.benchnark` symbols:
```
e: file:///Users/filipp.zhinkin/Development/kx-benchmark-android/examples/kotlin-multiplatform/src/androidMain/kotlin/AndroidTestBenchmark.kt:6:7 This class does not have a constructor
e: file:///Users/filipp.zhinkin/Development/kx-benchmark-android/examples/kotlin-multiplatform/src/androidMain/kotlin/AndroidTestBenchmark.kt:6:14 Unresolved reference: Benchmark
e: file:///Users/filipp.zhinkin/Development/kx-benchmark-android/examples/kotlin-multiplatform/src/androidMain/kotlin/AndroidTestBenchmark.kt:7:8 This class does not have a constructor
e: file:///Users/filipp.zhinkin/Development/kx-benchmark-android/examples/kotlin-multiplatform/src/androidMain/kotlin/AndroidTestBenchmark.kt:8:13 This class does not have a constructor
```

It does not make much sense to redefine a dependency for this sample project, but the same issue is also reproduced on `kotlinx-io` when I tried to apply kx-benchmark w/ android support.